tensorflow_datasets/core/download/download_manager.py

```diff
@@ -241,12 +249,22 @@ def __getstate__(self):
 
   @property
   def _downloader(self):
+    """Returns the downloader object.
```

Comment:
Here, a one-line docstring is nice, but typing annotation would be even more helpful

```py
@property
def _downloader(self) -> downloader.Downloader:
```
Is more helpful than
```py
@property
def _downloader(self):
"""Returns the downloader object."""
```

You don't have to change it there but in the future, adding typing annotation is more helpful than adding docstring.

Reply:
Sounds good.